hw03_barkovskan.py

```diff
@@ -0,0 +1,41 @@
+import turtle
+
+def drawsquare(bubbles, size):
```

Comment:
Pay attention to the gray squigglies, they are style guidelines. This one says two blank lines before all function definitions.